tests/components/conditional_formatting.test.ts

```diff
@@ -1123,6 +1123,18 @@ describe("UI of conditional formats", () => {
     expect(errorMessages()).toEqual(["Invalid Maxpoint formula"]);
   });
 
+  test("Hides the 'No Color' button when the color picker is opened for the color scale", async () => {
+    triggerMouseEvent(selectors.buttonAdd, "click");
```

Comment:
you can use the helper `simulateClick`, otherwise, seems good :)

Reply:
Suggested changes made :)